src/common.php

```diff
@@ -88,4 +106,47 @@ function linkToScratchProfile(string $username) : string {
 		],
 		$username
 	);
+}
+
+function blockExpirationForm($lang, $user, bool $showCurrent, ?string $current) : string {
```

Comment:
if I select "other time", the text input still seems to remain disabled on my local

Reply:
Probably a JS error. Try reloading?